tests/queryset/modify.py

```diff
@@ -94,6 +96,30 @@ def test_modify_with_fields(self):
         self.assertEqual(old_doc.to_mongo(), {"_id": 1})
         self.assertDbEqual([{"_id": 0, "value": 0}, {"_id": 1, "value": -1}])
 
+    @needs_mongodb_v26
+    def test_modify_with_push(self):
+        class BlogPost(Document):
+            id = StringField(primary_key=True)
```

Comment:
You can remove this line w/o any effect on the code - an id field will be created anyway.